src/databricks/labs/ucx/assessment/clusters.py

```diff
@@ -161,3 +161,55 @@ def snapshot(self) -> Iterable[ClusterInfo]:
     def _try_fetch(self) -> Iterable[ClusterInfo]:
         for row in self._fetch(f"SELECT * FROM {self._schema}.{self._table}"):
             yield ClusterInfo(*row)
+
+
+@dataclass
+class PolicyInfo:
+    policy_id: str
+    cluster_id: str
+    dbr_version: str
+    policy_name: str | None = None
+    creator: str | None = None
+
+
+class PoliciesCrawler(CrawlerBase[PolicyInfo], CheckClusterMixin):
+    def __init__(self, ws: WorkspaceClient, sbe: SqlBackend, schema):
+        super().__init__(sbe, "hive_metastore", schema, "policies", PolicyInfo)
+        self._ws = ws
+
+    def _crawl(self) -> Iterable[PolicyInfo]:
+        all_clusters = list(self._ws.clusters.list())
```

Comment:
List all policies, not clusters!

Reply:
Updated the code to list all policies instead of clusters. Since Policy won't be having DBR version associated with it I  have added the dbr version in cluster and joining the cluster with policies to get the required information

Reply:
policy can have `spark_version` in it: https://docs.databricks.com/en/administration-guide/clusters/policy-definition.html#supported-attributes